bundles/framework/divmanazer/instance.js

```diff
@@ -139,6 +140,16 @@ Oskari.clazz.define('Oskari.userinterface.bundle.ui.UserInterfaceBundleInstance'
                 me.requestHandlers.modal
             );
 
+            // Hide navigation if starts in mobile mode
+            const nav = Oskari.dom.getNavigationEl();
+            Oskari.dom.showNavigation(!Oskari.util.isMobile());
+            if (nav) {
+                const mapModule = this.sandbox.findRegisteredModuleInstance('MainMapModule');
```

Comment:
We only need to hide the navigation if it's in the DOM.

```suggestion
            const nav = Oskari.dom.getNavigationEl();
            if (nav) {
                Oskari.dom.showNavigation(!Oskari.util.isMobile());
                const mapModule = this.sandbox.findRegisteredModuleInstance('MainMapModule');
```